eng/targets/GenerateAnalyzerNuspec.targets

```diff
@@ -111,15 +110,11 @@
 
-    <Exec Command='"$(DotNetExecutable)" --roll-forward major "$(_GenerateDocumentationAndConfigFilesPath)" "-validateOnly:false" "$(_GeneratedRulesetsDir)" "$(_GeneratedEditorconfigsDir)" "$(_GeneratedGlobalAnalyzerConfigsDir)" "$(ArtifactsBinDir)$(EscapeDirectorySuffix)" "$(Configuration)" "%(AnalyzerRulesetAssembly.TargetFramework)" "@(AnalyzerRulesetAssembly)" "$(PackagePropsFileDir)" "$(PackagePropsFileName)" "$(PackageTargetsFileDir)" "$(PackageTargetsFileName)" "$(DisableNETAnalyzersPackagePropsFileName)" "$(AnalyzerSarifFileDir)" "$(AnalyzerDocumentationFileName)" "$(AnalyzerSarifFileDir)" "$(AnalyzerSarifFileName)" "$(VersionPrefix)" $(NuspecPackageId) $(ContainsPortedFxCopRules) $(GenerateAnalyzerRulesMissingDocumentationFile) $(ReleaseTrackingOptOut) $(_ValidateOffline)' />
+    <Exec Command='"$(DotNetExecutable)" --roll-forward major "$(_GenerateDocumentationAndConfigFilesPath)" --validateOnly false --analyzerRulesetsDir "$(_GeneratedRulesetsDir)" --analyzerEditorconfigsDir "$(_GeneratedEditorconfigsDir)" --analyzerGlobalconfigsDir "$(_GeneratedGlobalAnalyzerConfigsDir)" --binDirectory "$(ArtifactsBinDir)$(EscapeDirectorySuffix)" --configuration "$(Configuration)" --tfm "%(AnalyzerRulesetAssembly.TargetFramework)" --assemblies "@(AnalyzerRulesetAssembly)" --propsFileDir "$(PackagePropsFileDir)" --propsFileName "$(PackagePropsFileName)" --targetsFileDir "$(PackageTargetsFileDir)" --targetsFileName "$(PackageTargetsFileName)" --propsFileToDisableNetAnalyzers "$(DisableNETAnalyzersPackagePropsFileName)" --analyzerDocumentationFileDir "$(AnalyzerSarifFileDir)" --analyzerDocumentationFileName "$(AnalyzerDocumentationFileName)" --analyzerSarifFileDir "$(AnalyzerSarifFileDir)" --analyzerSarifFileName "$(AnalyzerSarifFileName)" --analyzerVersion "$(VersionPrefix)" --analyzerPackageName $(NuspecPackageId) --containsPortedFxcopRules $(ContainsPortedFxCopRules) --generateAnalyzerRulesMissingDocumentationFile $(GenerateAnalyzerRulesMissingDocumentationFile) --releaseTrackingOptOut $(ReleaseTrackingOptOut) --validateOffline $(ValidateAnalyzerDocsOffline)' />
```

Comment:
Rather than maintaining the argument list twice should we just set a property based on ContinuousIntegrationBuildCorrectness and pass that in? I realize this isn't new, but this is kinda a nightmare.

Reply:
So actually when `ContinuousIntegrationBuildCorrectness` is true, there are two runs (this one has no condition).  That is the same behavior as before - one with validation, one without.  
I'm not 100% sure why but I think the second one needs to actually generate some targets file?

I could extract it out to maybe its own target that I can pass in a value, or dig into having validate also generate, but imho the new version here is better enough for my timebox.